pandas/core/indexes/datetimes.py

```diff
@@ -1770,11 +1770,38 @@ def date(self):
 
     def normalize(self):
         """
-        Return DatetimeIndex with times to midnight. Length is unaltered
+        Return DatetimeIndex with times converted to midnight.
+
+        When using DatetimeIndex, the time can be converted to midnight i.e.
+        00:00:00. This is useful in cases, when the time does not matter.
+        Length is unaltered. The timezones are unaffected. If the frequency
```

Comment:
I would remove the last sentence. freq is actually re-inferred here, so this is misleading.